rcon/utils.py

```diff
@@ -382,22 +390,37 @@ def __len__(self):
         return self.red.llen(self.key)
 
 
-class MapsHistory(FixedLenList):
+class PlayerStat(TypedDict):
+    combat: int
+    offense: int
+    defense: int
+    support: int
+
+
+class MapInfo(TypedDict):
+    name: str
+    start: None | float
+    end: None | float
+    guessed: bool
+    player_stats: dict[str, PlayerStat]
+
```

Comment:
Think you could add these to `rcon/types` so we have all (or most of) our typedefs in one spot?